ffs/api/api_paych.go

```diff
@@ -0,0 +1,50 @@
+package api
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ipfs/go-cid"
+	"github.com/textileio/powergate/ffs"
+)
+
+// ListPayChannels lists all payment channels associated with this FFS
+func (i *API) ListPayChannels(ctx context.Context) ([]ffs.PaychInfo, error) {
```

Comment:
The ffs pay channel api is mostly verifying that the addresses involved in payment channel operations are managed by the ffs instance, then calling down into the `paych` module.